content/pages/why.md

```diff
@@ -0,0 +1,89 @@
+Title: Why SaltyRTC?
+Date: 2016-12-03 19:00
+Modified: 2016-12-03 19:00
+Sort-Order: 1
+Summary: What is SaltyRTC all about and why should I use it?
+
+# Short
+
+Here's why you should use SaltyRTC. With SaltyRTC,
+
+* signalling data is fully end-to-end encrypted,
+* zero trust towards the server is needed,
+* no data is stored on the server,
+* servers can be self-hosted,
```

Comment:
With self-hosting you mean that people can put it on their own server, right? Doesn't that apply to most signaling server implementations (except the SAAS-type)?